ansible/library/sensors_facts.py

```diff
@@ -107,12 +107,13 @@ def collect_sensors(self):
             process = subprocess.Popen(['sensors', '-A', '-u'], stdout=subprocess.PIPE, stdin=subprocess.PIPE)
```

Comment:
Actually, there should be a parameter setting `stderr=subprocess.PIPE` here. `stdin` doesn't need to be set to `PIPE` here; the default value of `None` is fine, since there's no input needed into the process.